prime-dotnet-webapi/Services/AccessTermService.cs

```diff
@@ -64,12 +64,14 @@ public async Task CreateEnrolleeAccessTermAsync(Enrollee enrollee)
 
         public async Task SetAcceptedDateForAccessTermAsync(Enrollee enrollee)
```

Comment:
Probably should rename this method, it doesn't do what it says any more. Perhaps something like AcceptCurrentAccessTerm